src/gt4py/next/iterator/embedded.py

```diff
@@ -182,9 +183,12 @@ def field_setitem(self, indices: FieldIndexOrIndices, value: Any) -> None:
         ...
 
 
-# TODO see https://github.com/GridTools/gt4py/pull/1120
-_column_range: Optional[range] = None
-_offset_provider: Optional[OffsetProvider] = None
+#: Column range used in column mode (`column_axis != None`) in the current closure execution context.
+context_column_range: cvars.ContextVar[range] = cvars.ContextVar("context_column_range")
```

Comment:
I‘m not convinced of the naming scheme, but your choice.

Reply:
Any suggestion?

Reply:
Changed